crates/core/tedge/src/cli/reconnect/cli.rs

```diff
@@ -0,0 +1,45 @@
+use crate::cli::common::Cloud;
+use crate::cli::connect::CommonMosquittoConfig;
+use crate::command::*;
+use tedge_config::system_services::service_manager;
+
+use super::command::ReconnectBridgeCommand;
+
+const C8Y_CONFIG_FILENAME: &str = "c8y-bridge.conf";
+const AZURE_CONFIG_FILENAME: &str = "az-bridge.conf";
+
+#[derive(clap::Subcommand, Debug)]
+pub enum TEdgeReconnectBridgeCli {
```

Comment:
```suggestion
pub enum TEdgeReconnectCli {
```

As we're not just reconnecting the bridge but stopping/starting other services as well.

Reply:
Done